cartridges/int_adyen_controllers/cartridge/scripts/payment/processor/ADYEN_CREDIT.js

```diff
@@ -126,11 +109,11 @@ function Authorize(args) {
             view: app.getView({
                 ContinueURL: URLUtils.https('Adyen-CloseIFrame', 'utm_nooverride', '1'),
                 Basket: order,
-                issuerUrl: result.IssuerUrl,
-                paRequest: result.PaRequest,
-                md: result.MD
+                issuerUrl : result.RedirectObject.url,
+                paRequest : result.RedirectObject.data.PaReq,
```

Comment:
I think it can happen that the data is missing from the response object - should be checked before